src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstance.kt

```diff
@@ -21,19 +23,34 @@ class LocalInstance private constructor(aem: AemExtension) : AbstractInstance(ae
     override lateinit var password: String
 
     var debugPort: Int = 5005
+    var debugAddress: String = ""
```

Comment:
DebugHost

Reply:
it is named as `address` in Java options, I want to stay consistent with Jva

Reply:
To truly follow standards I can name it Socket Address: https://networkengineering.stackexchange.com/questions/9429/what-is-the-proper-term-for-ipaddress-hostnameport

:)

Reply:
it can be both IP and host name, so debug host will be misleading